app/models.py

```diff
@@ -2004,6 +2020,17 @@ class AliasMailbox(db.Model, ModelMixin):
     alias = db.relationship(Alias)
 
 
+class AliasHibp(db.Model):
```

Comment:
I think we can make this class inherit from `ModelMixin` too so it can have some fields and utility methods. Almost all model classes inherit from ModelMixin I think :).